pre_commit_hooks/pretty_format_json.py

```diff
@@ -17,36 +15,38 @@ def pairs_first(pairs):
         if sort_keys:
             after = sorted(after, key=lambda x: x[0])
         return OrderedDict(before + after)
-    return six.text_type(simplejson.dumps(
-        simplejson.loads(
-            contents,
-            object_pairs_hook=pairs_first,
-        ),
+    json_pretty = json.dumps(
+        json.loads(contents, object_pairs_hook=pairs_first),
         indent=indent,
         ensure_ascii=ensure_ascii,
-    )) + "\n"  # dumps does not end with a newline
+    )
+    # Ensure unicode under Py2
+    if sys.version_info.major == 2:
```

Comment:
will need a `no cover` comment to satisfy coverage.

Since six is still used, probably just use `six.PY2` here